projects/layout/components/card/collapsed.style.less

```diff
@@ -3,8 +3,21 @@
 [tuiCardCollapsed] {
     .transition(~'clip-path, margin');
 
-    border-image: radial-gradient(at center, var(--tui-background-neutral-2) 26.5%, transparent 27%) 50% 49% / 0 2rem
-        2rem / 0 0 2rem;
+    overflow: visible;
```

Comment:
I would like to fix this without changing the overflow rule, please do not merge this until I return so I can try to do that.